starter/source/materials/fail/nloc_dmg_init.F

```diff
@@ -244,6 +244,14 @@ SUBROUTINE NLOC_DMG_INIT(ELBUF_TAB,NLOC_DMG ,IPARG    ,IXC      ,
         ALLOCATE( TAGTET(NUMELS) )
         ALLOCATE( INDEX(NUMELS+NUMELC+NUMELTG) )
         ALLOCATE( ITRI(NUMELS+NUMELC+NUMELTG) )
+C
+        IF (NSUBDOM > 0) THEN
+C--       multidomains - origianl nummat is used
+          MATSIZE = NUMMAT0
+        ELSE
+          MATSIZE = NUMMAT
+        ENDIF
+        MY_ALLOCATE2D(WARN_LENGHT,MATSIZE,3)
```

Comment:
@bertrand-altair 

WARN_LENGHT is allocated here , but never deallocated

Reply:
fixed